Sources/Extensions/SwiftStdlib/DictionaryExtensions.swift

```diff
@@ -110,6 +110,24 @@ public extension Dictionary {
 
 }
 
+// MARK: - Methods (Value: Equatable)
+public extension Dictionary where Value: Equatable {
+
+    /// SwifterSwift: Returns an array of all keys that have the given value in dictionary.
+    ///
+    ///        let dict = ["key1": "value1", "key2": "value1", "key3": "value2"]
+    ///        dict.keys(forValue: "value1") -> ["key1", "key2"]
+    ///        dict.keys(forValue: "value2") -> ["key3"]
+    ///        dict.keys(forValue: "value3") -> []
+    ///
+    /// - Parameter forValue: Value for which keys are to be fetched.
```

Comment:
The parameter is actually `value`. You can always ensure you have the right documentation by putting the text cursor on the function name and selecting *Editor -> Structure -> Add Documentation*